app/pages/project/metadata.cjsx

```diff
@@ -1,12 +1,31 @@
 React = require 'react'
 {Link} = require 'react-router'
+visible = (require 'visible-element')()
 
 module?.exports = React.createClass
   displayName: 'ProjectMetadata'
 
   propTypes:
     project: React.PropTypes.object
 
+  contextTypes:
+    geordi: React.PropTypes.object
+
+  scrollDone: false
+
+  handleScroll: ->
+    return if @scrollDone
+
+    @scrollDone = visible.inViewport(@refs.belowStats)
+    @context.geordi?.logEvent type: 'scroll-down' if @scrollDone
+
+  componentDidMount: ->
```

Comment:
Per my comment about drag/resize above, I'm not sure we should be capturing scroll events across the board, only for specific experiments/situations. it will generate too much data.